buildenv/jenkins/Jenkinsfile_personal

```diff
@@ -0,0 +1,95 @@
+#!groovy
+pipeline {
+    agent {label "${params.MACHINE_LABEL}"}
+    parameters {
+		choice (choices: 'x64_linux\nx64_mac\ns390x_linux\nppc64le_linux\naarch64_linux', description: 'PLATFORM?', name: 'PLATFORM')
+		choice (choices: 'SE90\nSE80', description: 'What is JAVA_VERSION?', name: 'JAVA_VERSION')
+		choice (choices: 'openjdk9-openj9\nopenjdk8-openj9\nopenjdk9\nopenjdk8', description: 'What is JVM Version?', name: 'JVM_VERSION')
+		choice (choices: 'openjdk\nsanity\nextended\ntest_systemtest\nruntest', description: 'What is test level?', name: 'TARGET')
+		string (defaultValue: 'linux&&x64&&test', description: 'What is machine label?', name: 'MACHINE_LABEL')
```

Comment:
This could almost be auto-filled based on PLATFORM selection... (except that the labels may be different at different Jenkins servers..., so we can leave it as an explicitly filled out choice.

Reply:
updated.